hadoop-hdds/interface-client/src/main/proto/DatanodeClientProtocol.proto

```diff
@@ -250,7 +253,8 @@ message ContainerCommandResponseProto {
 
   optional   FinalizeBlockResponseProto finalizeBlock = 22;
   optional   EchoResponseProto echo = 23;
-}
+  optional   ReadBlockResponseProto readBlock = 24;
```

Comment:
Do you have to have two different types of responses? Can't the Datanode just send `StreamDataResponseProto` repeatedly for the range requested?

Reply:
Done. Now, there is one type `ReadBlockResponseProto`, it is for collecting repeatedly `readChunkResponseProto` to fit in the current class `XceiverClientReply`.